### PR TITLE
fix: plugins with new usages of projects and assets

### PR DESCRIPTION
**Pull request by hguerni**

### CODE REVIEW

File: recipes/datasets/plugins/parallel_labeling_plugin.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ def on_submit(self, label: Label, asset_id: str) -> None:
                 "jsonResponse",
                 "isLatestLabelForUser",
             ],
+            as_generator=True,
```
[the remark this answers is not in the document]
Comment (hguerni) [reply recipes/datasets/plugins/parallel_labeling_plugin.py:67]: it is to control the type return of the labels overload

[the remark this answers is not in the document]
Comment (gheorghetutunaru) [reply recipes/datasets/plugins/parallel_labeling_plugin.py:67]: Yes, but why does it need to be a generator and not the default list (as done for the `kili.projects` call) ?

[the remark this answers is not in the document]
Comment (gheorghetutunaru) [reply recipes/datasets/plugins/parallel_labeling_plugin.py:67]: You resolved the conversation, but did not actually answer

[the remark this answers is not in the document]
Comment (hguerni) [reply recipes/datasets/plugins/parallel_labeling_plugin.py:67]: As an example it seemed fitting to me to have both type of call, giving both type of return, presented

[the remark this answers is not in the document]
Comment (hguerni) [reply recipes/datasets/plugins/parallel_labeling_plugin.py:67]: The generator used to be the default return

[the remark this answers is not in the document]
Comment (hguerni) [reply recipes/datasets/plugins/parallel_labeling_plugin.py:67]: I thought the question was about the `as_generator` attribute, sorry I closed the conversation early

**Review by florianlega — APPROVED** [review]

File: recipes/datasets/plugins/parallel_labeling_plugin.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (pmKili) [reply recipes/datasets/plugins/parallel_labeling_plugin.py:67]: @hguerni I still don't understand why you need the as_generator

**Pull request closed without merging**